doc/classes/EditorDock.xml

```diff
@@ -67,6 +67,12 @@
 				Closes the dock, making its tab hidden.
 			</description>
 		</method>
+		<method name="make_visible">
+			<return type="void" />
+			<description>
+				Focuses the dock's tab (or window if it's floating). If the dock was closed, it will be opened. If it's a bottom dock, makes the bottom panel visible.
```

Comment:
Believe it or not we rarely use "focus" as an actual verb. Oftentimes it's something like "gain/grab focus", "receive the focus" "has focus", etc.
However, there is one occurrence of "open and focus the dock" in `add_control_to_dock` in EditorDock, so maybe that's fine...